google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -767,13 +752,15 @@ def export_images(
         output_filenames: List[str] = []
         index = 0
         for entity in self.entities:
-            if not entity.image:
+            image = entity.crop_image(
+                documentai_page=self.pages[entity.start_page].documentai_page
+            )
+            if not image:
                 continue
-
             output_filename = (
                 f"{output_file_prefix}_{index}_{entity.type_}.{output_file_extension}"
```

Comment:
Update the docstring to include the output filename format.

Reply:
This is already in the docstring for the function. Am I missing something?